pkgs/development/ocaml-modules/angstrom-async/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, buildDunePackage, angstrom, async }:
+
+buildDunePackage rec {
+  pname = "angstrom-async";
+  version = "0.12.1";
+
+  minimumOCamlVersion = "4.04.1";
+
+  src = fetchFromGitHub {
```

Comment:
You can “inherit” this attribute and a few other ones (“version”, most of “meta”) from the “angstrom” derivation.

Reply:
Done.